src/utils/vscodeUtils.ts

```diff
@@ -78,3 +81,15 @@ export function fetchNodeModule(moduleName: string) {
     } catch (err) { }
     return null;
 }
+
+export function getLabel(node: IAzureNode): string {
+    let labels = [node.treeItem.label];
+    while (node.parent) {
+        node = node.parent;
+        labels.unshift(node.treeItem.label);
+        if ((node.treeItem instanceof MongoAccountTreeItem) || (node.treeItem instanceof DocDBAccountTreeItemBase)) {
```

Comment:
Extract into a isAccountTreeItem() method

Reply:
Ideally we'd have something more generic but not sure it's worth it for this small case.